website/docs/components/form/masked-input/partials/code/how-to-use.md

```diff
@@ -0,0 +1,253 @@
+!!!
+
+There are two ways to use the Masked Input component:
+
+- `Form::MaskedInput::Base` - the base component: the input control with a toggle button.
+- `Form::MaskedInput::Field` - the field component: the input control with a toggle button, a label, helper text, and error messaging (in a wrapping container).
+
+We recommend using the Field component as it provides built-in accessibility functionality. Use the Base component if needing to achieve custom layouts or for special use cases not covered by the Field component.
+
+{{! ================= }} {{! ===== FIELD ===== }} {{! ================= }}
+
+### Form::MaskedInput::Field
+
+The basic invocation requires a `Label`. This creates:
+
+- a `<label>` element with a `for` attribute automatically associated with the input `ID` attribute.
+- a `<input type="text">` or a `<textarea>` control with an automatically generated `ID` attribute.
+
+```handlebars
+<Hds::Form::MaskedInput::Field as |F|>
+  <F.Label>Terraform Cloud team token</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+#### Input value
+
+Pass a `@value` argument to pre-populate the input. By default, the content is visually obfuscated ("masked") and users can make it visible using the associated toggle button.
+
+```handlebars
+<Hds::Form::MaskedInput::Field @value="036215df4996ca649928d8864b4df9e42cba0d6d" as |F|>
+  <F.Label>Terraform Cloud team token</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+If you need to make the content visible by default or control the masking from outside the component use the `@isMasked` argument.
+
+```handlebars
+<Hds::Form::MaskedInput::Field @isMasked={{false}} @value="036215df4996ca649928d8864b4df9e42cba0d6d" as |F|>
+  <F.Label>Terraform Cloud team token</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+#### Multiline
+
+Set `@isMultiline` argument to `true` to render a `<textarea>`
+
+```handlebars
+<Hds::Form::MaskedInput::Field @isMultiline={{true}} @value="-----BEGIN RSA PRIVATE KEY-----
+MIIBOgIBAAJBAKj34GkxFhD90vcNLYLInFEX6Ppy1tPf9Cnzj4p4WGeKLs1Pt8Qu
+KUpRKfFLfRYC9AIKjbJTWit+CqvjWYzvQwECAwEAAQJAIJLixBy2qpFoS4DSmoEm
+o3qGy0t6z09AIJtH+5OeRV1be+N4cDYJKffGzDa88vQENZiRm0GRq6a+HPGQMd2k
+TQIhAKMSvzIBnni7ot/OSie2TmJLY4SwTQAevXysE2RbFDYdAiEBCUEaRQnMnbp7
+9mxDXDf6AU0cN/RPBjb9qSHDcWZHGzUCIG2Es59z8ugGrDY+pxLQnwfotadxd+Uy
+v/Ow5T0q5gIJAiEAyS4RaI9YG8EWx/2w0T67ZUVAw8eOMB6BIUg0Xcu+3okCIBOs
+/5OiPgoTdSy7bcF9IGpSE8ZgGKzgYQVZeN97YE00
+-----END RSA PRIVATE KEY-----" as |F|>
+  <F.Label>Private key</F.Label>
+</Hds::Form::MaskedInput::Field>
+```
+
+!!! Info
+
+**Important to know**
+
+When the multiline input is masked, the browser converts newline characters to masked characters: this means that the multiline text will appear as a single long string of characters, even if it's inside a `<textarea>` element.
+
+Instead, when the text is not masked it will respect the newline characters: this means it may occupy more lines that when it's masked (try the example above).
```

Comment:
```suggestion
Instead, when the text is not masked it will respect the newline characters: this means it may occupy more lines that when it’s masked (try the example above).
```